src/test/java/com/automatedtest/sample/definitions/ImageSearchPageSteps.java

```diff
@@ -0,0 +1,38 @@
+package com.automatedtest.sample.definitions;
+
+import com.automatedtest.sample.pages.ImageSearchPage;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+public class ImageSearchPageSteps {
```

Comment:
in page object model, we should not create a page just to store some steps from another page. If all of this actions is on Search page, then just put it on SearchResultPage